packages/react-reconciler/src/ReactFiberClassComponent.new.js

```diff
@@ -711,8 +711,7 @@ function callComponentWillMount(workInProgress, instance) {
 
   if (typeof instance.componentWillMount === 'function') {
     instance.componentWillMount();
-  }
-  if (typeof instance.UNSAFE_componentWillMount === 'function') {
+  }else if (typeof instance.UNSAFE_componentWillMount === 'function') {
```

Comment:
```suggestion
  } else if (typeof instance.UNSAFE_componentWillMount === 'function') {
```

To match the existing code style:

https://github.com/facebook/react/blob/4c7036e807fa18a3e21a5182983c7c0f05c5936e/packages/react-reconciler/src/ReactFiberClassComponent.new.js#L559-L560

---

This can potentially be applied to [`ReactFiberClassComponent.old.js`](https://github.com/facebook/react/blob/4c7036e807fa18a3e21a5182983c7c0f05c5936e/packages/react-reconciler/src/ReactFiberClassComponent.old.js#L764-L769) as well.